src/views/dashboard/status/index.vue

```diff
@@ -1,37 +1,58 @@
 <template lang="pug">
+a-modal.guide-modal(
+  v-model:visible="statusModal"
+  :mask-closable="false"
+  :ok-text="$t('status.confirm')"
+  :hide-cancel="true"
+  :closable="false"
+  @ok="handleOk"
+)
+  template(#title)
+     div {{ $t('status.info') }}
+     svg.guide-banner
+      use(href="#banner")
+
 a-layout.layout
   a-layout-content
-    a-card(title="GreptimeDB Status")
+
+    a-card(title='GreptimeDB Status')
       template(#extra)
         a-button(type="text" @click="refreshStatus") refresh
-      a-empty(v-if="!statusInfoRef")
-        template(#image)
-          svg.icon-32
-            use(href="#empty")
-        | Status is not supported until GreptimeDB v0.3.1
-      a-descriptions(v-else bordered :column="2")
-        a-descriptions-item(v-for="item of statusInfoRef" :label="item[0]")
+        a-button(type="text" @click="copyToClipboard") copy to clipboard
+      a-descriptions(:column='2' bordered)
+        a-descriptions-item(v-for='item of statusInfoRef', :label='item[0]')
```

Comment:
It would be better if you preserve the original `a-empty` component. Those who are using GreptimeDB v0.3.0 and under, couldn't see the status on this page. So we want to give the user an empty status for clarification.